torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp

```diff
@@ -890,12 +890,35 @@ void propagateQuantizationOps(Block* block) {
         }
       }
     } else {
+      // For this category of ops, we need to
```

Comment:
Also, when you mean "this category" of ops, do you mean operations that have multiple inputs or outputs?.

Reply:
no, this is for ops that we quantize by propagating dequantize ops, e.g. flatten

Reply:
> no, this is for ops that we quantize by propagating dequantize ops, e.g. flatten

maybe we can include this in the comment?